styles/main.css

```diff
@@ -130,7 +132,7 @@ footer hr {
     border-top-right-radius: 8px;
 }
 
-.mugshot ~ * {
+* {
     box-sizing: border-box;
 }

```

Comment:
The selector you have here doesn't apply this rule to ALL elements.  Leaving it as just `*` does so.